lib/faker/default/chile_rut.rb

```diff
@@ -92,6 +92,29 @@ def full_rut(legacy_min_rut = NOT_GIVEN, legacy_fixed = NOT_GIVEN, min_rut: 0, f
         "#{rut(min_rut: min_rut, fixed: fixed)}-#{dv}"
       end
 
+      ##
+      # Produces a random Chilean RUT (Rol Unico Tributario, ID with 8 digits) with a dv (digito verificador, check-digit).
+      # with character passed in argument as separator.
+      #
+      # @param min_rut [Integer] Specifies the minimum value of the rut.
+      # @param fixed [Boolean] Determines if the rut is fixed (returns the min_rut value).
+      # @return [String]
+      #
+      # @example
+      #   Faker::ChileRut.full_rut_with_dots #=> "30.686.957-4"
+      #   Faker::ChileRut.full_rut_with_dots(min_rut: 20890156) #=> "30.686.957-4"
+      #   Faker::ChileRut.full_rut_with_dots(min_rut: 30686957, fixed: true) #=> "30.686.957-4"
+      #
+      # @faker.version 1.9.2
+      def full_rut_with_dots(legacy_min_rut = NOT_GIVEN, legacy_fixed = NOT_GIVEN, min_rut: 0, fixed: false)
```

Comment:
As the purpose of this method is to return a formatted Rut number, I'd suggest considering using the `full_formatted_rut` name.

```suggestion
      def full_formatted_rut(legacy_min_rut = NOT_GIVEN, legacy_fixed = NOT_GIVEN, min_rut: 0, fixed: false)
```

Reply:
Thanks for your review @vbrazo , you're right, it is a more meaningfull name. I've changed the name and updated the test.